### PR TITLE
Docs/lk exploration [SLT-170]

### PR DESCRIPTION
**Pull request by lawsonkight**
First pass at basic styling and content management for Synapse Docs.

Creates a new /docs/Overview page summarizing the sidebar content.

Proposal: top-level `Routers` folder for CCTP and RFQ content
Proposal: rename `Services` to `Essential Tools`
Proposal: add `Observability` to `Essential Tools` as `Telemetry`

**Metadata**
- https://www.notion.so/distributedsystems/Docs-Design-10324013016080fcac76e20878d10b06?pvs=4

## Summary by CodeRabbit

- **New Features**
  - Introduced comprehensive documentation for the CCTP Router, detailing the native minting and burning of USDC.
  - Added visual representations of the CCTP flow for better understanding of asset transfers.
  - Included instructions for configuring the CCTP Relayer with example YAML configurations.

- **Documentation**
  - Updated documentation on the interaction between Synapse CCTP contracts and Circle CCTP contracts.
  - Provided operational instructions for running the relayer using various methods, including Docker and Helm.
  - Updated dependencies in the documentation project for improved performance and compatibility.

c444c64f6ee913070c92a80d4e14d90b3a3da3b5: [docs preview link ](https://bridge-docs-45wqit1d1-synapsecns.vercel.app)
832ce5cd8fb54d8a47c2ecd7c6e91d68a56d0469: [docs preview link ](https://bridge-docs-5rsycog7k-synapsecns.vercel.app)
1f4d74793b008f197c02060f5581e7c2b1bcb00f: [docs preview link ](https://bridge-docs-oeazdru0v-synapsecns.vercel.app)
4571733ef87fac08a464930f0ff91bcd5a9b89d5: [docs preview link ](https://bridge-docs-5g6pgafce-synapsecns.vercel.app)
6e05bf62263a015fa999cb881bc24d986ded38be: [docs preview link ](https://bridge-docs-6apo9w6da-synapsecns.vercel.app)
bad6118de256d1f686b21e5ebd68696671e09d00: [docs preview link ](https://bridge-docs-1u54rr0tn-synapsecns.vercel.app)
9844b73952afd63b4c50313c52dae040a0d423f2: [docs preview link ](https://bridge-docs-aoiurd8xb-synapsecns.vercel.app)
61c8cfeb6e2fd82b26685c58fb403d148f2a7bf4: [docs preview link ](https://bridge-docs-6gsubl0hi-synapsecns.vercel.app)
1f30f7eb477cd028bb939f4bce52eab6c9fb7a92: [docs preview link ](https://bridge-docs-849jr31ld-synapsecns.vercel.app)
fc6eeb83a944bb877885215e3cf785090a3189c6: [docs preview link ](https://bridge-docs-b7o2hjcsn-synapsecns.vercel.app)
ef32561e0ad1a49f4ed9492037e0c72e271eae9d: [docs preview link ](https://bridge-docs-gcj12koap-synapsecns.vercel.app)

### CODE REVIEW

**Review by trajan0x — CHANGES_REQUESTED** [review]
One other thing I'd like us to consider is 301 redirects for old docusaudus page routes, at least important ones

Not blocking

File: docs/bridge/docs/01-About/01-DAO.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Governance Model
+
+SYN holders with more than 100,000 SYN tokens can put forward proposals to be be voted on by the DAO. SYN holders can vote using SYN on multiple chains, or delegate their votes to other token holders. 
+
+In order to be adopted by the DAO, a proposal must gather at least `50% + 1` of all votes and reach the minimum quorum of 5 million SYN tokens.
+
+## Governance Components
+
+* [Forum](https://forum.synapseprotocol.com/): A Platform to post proposals for discussion before an official vote
+* [Snapshot](https://snapshot.org/#/synapseprotocol.eth): A decentralized governance platform for conducting official votes.
```
[the remark this answers is not in the document]
Comment (Defi-Moses) [reply docs/bridge/docs/01-About/01-DAO.md:18]: We can link to discord here too

File: docs/bridge/docs/01-About/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    </figure>
+</figure>
+
+* **[Synapse Router](/docs/Routers/Synapse-Router)** – Executable quotes for arbitrary blockchain transactions.
+* **[CCTP](/docs/Routers/CCTP)** – Native router for USDC transactions.
+* **[RFQ](/docs/Routers/RFQ)** – Fast router that allows on-chain agents to bid on interchain delivery.
+
+## Essential Services
+
+Bolt-on services for reliability and ease-of-use:
```
[the remark this answers is not in the document]
Comment (Defi-Moses) [reply docs/bridge/docs/01-About/index.md:61]: Agreed, these are not user facing nor are important to someone interfacing with the protocol.

**Comment by Defi-Moses** [comment]
Some specific feedback. Happy to elaborate on any specific part.

_Home page:_

- [ ] add  “analytics” to the synapse explorer... I want to communicate that people can find protocol stats here too

_API:_

- [ ] point to swagger docs for more robust information. api.synapseprotocol.com/api-docs (This is what developers will be looking for/ want to reference)
- [ ] Add extra optional variable to /bridge and /bridgeTxInfo that we just added today (I will handle)
- [ ] change example addresses to something else (accidentally used an existing users) -- Ill also change

_Synapse Router:_

- [ ]  “most transactions require an intermediary token” is no longer true, ~90% of volume and > 80% dont require intermediate tokens anymore, can slightly change the words here

_CCTP:_

- [ ] We should be a bit more specific with what users can do witht he CCTP relayer, what it does. The code snippets show how to run it but its not very clear what is being run/what it does. 
- [ ] Lets add Uniswap to the list of pools supported (think it has a bigger brand [https://github.com/synapsecns/synapse-contracts/tree/885cbe06a960591b1bdef330f3d3d57c49dba8e2/contracts/router/modules/pool/uniswap]

_Contracts:_
**Synapse router:**
- [ ] We include the bridge addresses at the beginning, we should clarify that the router addresses route through the bridge contract (which is actually doing the bridging)
**Liquidity Pools**
- [ ] nUSD/nETH are not liquidity pools in the contracts portion. Maybe they fit in neatly here but they are tokens

CCTP and RFQ contracts
- [ ] Again should clarify the difference between the router and the bridge contracts.


Going to check some of these boxes myself but I think these are in a great state

**Pull request merged**